src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -26,27 +26,40 @@ function clone<T extends GPUBindGroupLayoutDescriptor>(descriptor: T): T {
 
 export const g = makeTestGroup(ValidationTest);
 
-g.test('some_binding_index_was_specified_more_than_once').fn(async t => {
-  const goodDescriptor = {
-    entries: [
-      { binding: 0, visibility: GPUShaderStage.COMPUTE, type: 'storage-buffer' as const },
-      { binding: 1, visibility: GPUShaderStage.COMPUTE, type: 'storage-buffer' as const },
-    ],
-  };
-
-  // Control case
-  t.device.createBindGroupLayout(goodDescriptor);
+g.test('duplicate_bindings')
+  .desc('Test that uniqueness of binding numbers across entries is enforced.')
+  .subcases(() =>
+    poptions('bindings', [
+      { numbers: [0, 1], valid: true },
+      { numbers: [0, 0], valid: false },
+    ])
+  )
```

Comment:
nit: you can write this as:

```suggestion
  .subcases(() => [
    { bindings: [0, 1], _valid: true },
    { bindings: [0, 0], _valid: false },
  ])
```

Reply:
(with appropriate changes below)

Reply:
Applied suggestion in https://github.com/gpuweb/cts/commit/f623e9fb9c4798df210b7c6812eef98290105082